pkg/virtctl/usbredir/usbredir_default.go

```diff
@@ -61,9 +72,15 @@ func Run(cmd *cobra.Command, args []string) error {
 		}
 	}(cancelFn)
 
-	if usbredirClient, err := NewUSBRedirClient(ctx, "localhost:0", usbredirVMI); err != nil {
+	usbredirClient, err := NewUSBRedirClient(ctx, "localhost:0", usbredirVMI)
+	if err != nil {
 		return fmt.Errorf("Can't create usbredir client: %s", err.Error())
-	} else {
-		return usbredirClient.Redirect(usbdeviceArg)
 	}
+
+	if disableLaunch {
+		// This is a log to the user, should use stdout instead of default stderr of log
+		fmt.Fprintf(cmd.OutOrStdout(), "User can connect usbredir client at: %s\n", usbredirClient.GetProxyAddress())
```

Comment:
You could use `cmd.Printf` here

Reply:
that's way better, didn't see this one. Thanks.